mivs/templates/mivs_admin/game_results.html

```diff
@@ -56,9 +60,18 @@
         <td>{{ review.game_content_bad|yesno }}</td>
         <td>{% if review.game_score %}{{ review.game_score }}{% endif %}</td>
         <td>{{ review.game_review|linebreaksbr }}</td>
+    <td><input type="checkbox" name="review_id" value="{{ review.id }}"
+               {% if review.send_to_studio %} checked {% endif %}/></td>
     </tr>
 {% endfor %}
 </tbody>
 </table>
+    <button type="submit" class="btn btn-primary"
+            {% if game.status not in c.FINAL_GAME_STATUSES %} disabled title="You can only send reviews after a game has been accepted, waitlisted, or declined!"{% endif %}>
+      Mark Reviews for Sending to Indie Studio</button>
+  <p class="help-block">Please note that once the "Summary of judging results for your game" email is
```

Comment:
Hopefully this help text will allow any admin to understand how this feature works.

I wanted to make sure an admin couldn't accidentally sending multiple emails to the same studio, since that would be pretty awkward. I did this by making the email an Automated email -- those emails get sent once their criteria are met and will not be sent a second time. Unfortunately, the best criteria I could come up with is "has reviews waiting to send", meaning that once ANY reviews are ready to send the email will go out.

@LowPolyCount My recommended workflow is to simply wait until all the reviews are marked that you want sent out to folks, then approve the email. This should make sure that the emails are all correct and there's no mistakes when approving reviews to be sent to studios (e.g., you can double- and triple-check that all reviews marked for sending are appropriate for the studio to view).

Reply:
looks good to me!